src/components/Zoom/Zoom.js

```diff
@@ -90,7 +96,26 @@ function Zoom({
     [map],
   );
 
+  const zoomInDisabled = useMemo(() => {
+    return (
+      map.getView().getZoom() >=
+      map.getView().getConstrainedZoom(map.getView().getMaxZoom())
+    );
+  }, [currentZoom]);
+
+  const zoomOutDisabled = useMemo(() => {
+    return (
+      map.getView().getZoom() <=
```

Comment:
you can use currentZoom here
```suggestion
      currentzoom <=
```

Reply:
Tests were failing when I used currentZoom. I'll have another look